CONTRIBUTING.md

```diff
@@ -1,7 +1,73 @@
+# Rationale
+
+Contributions are always welcome. We accept [bug fixes](#bugs),
+[proposals and new ideas](#proposal), or simply [editorial
+improvements](#editorial-improvements).
+
+Please not that by submitting a pull request you are implicitly
+adhering to the [contributions greement](#contributions-agreement)
+
+## Bugs
+
+You can report a bug by creating an
+[issue](https://github.com/ARM-software/acle/issues). You cal also fix
+it yourself with a [pull
+request](https://github.com/ARM-software/acle/pulls).
+
+## Proposals
+
+We are open to any proposal, including the introduction of new sets of
+intrinsics. We _do not_ require to assign copyright to Arm for the
+introduction of new parts of the specifications. The original
```

Comment:
`the` is singular and `specifications` is plural, so 'the specification' or just 'specifications'.

Reply:
I agree my given rationale wasn't right. The reason it triggered my brain was because of the presence of two plurals: 'new parts' and 'specifications'. I looked it up and actually think it might have been OK, though it may read more naturally with only one. (I see you simplified it anyway 👍 ). Struggling to justify why though, maybe it's that specification should be singular in this context?